tests/integration/IntegrationTestCase.php

```diff
@@ -8,20 +8,20 @@
 
 class IntegrationTestCase extends TestCase
 {
-    protected function markAsSkippedIfMatejIsNotAvailable(): void
+    protected static function markAsSkippedIfMatejIsNotAvailable(): void
     {
         if (!getenv('MATEJ_TEST_ACCOUNTID')) {
-            $this->markTestSkipped('Environment variable MATEJ_TEST_ACCOUNTID has to be defined');
+            self::markTestSkipped('Environment variable MATEJ_TEST_ACCOUNTID has to be defined');
         }
 
         if (!getenv('MATEJ_TEST_APIKEY')) {
-            $this->markTestSkipped('Environment variable MATEJ_TEST_APIKEY has to be defined');
+            self::markTestSkipped('Environment variable MATEJ_TEST_APIKEY has to be defined');
         }
     }
 
-    protected function createMatejInstance(): Matej
+    protected static function createMatejInstance(): Matej
```

Comment:
If you declare the method static, you should use it via static call eveywhere where it is being used (in all other testcases)... Like `static::createMatejInstance()`

Reply:
:+1: fixed, thanks :)